doc/classes/Vector3i.xml

```diff
@@ -58,6 +65,36 @@
 				Returns a new vector with all components clamped between the components of [code]min[/code] and [code]max[/code], by running [method @GlobalScope.clamp] on each component.
 			</description>
 		</method>
+		<method name="cross" qualifiers="const">
+			<return type="Vector3i" />
+			<argument index="0" name="with" type="Vector3i" />
+			<description>
+				Returns the cross product of this vector and [code]b[/code].
+			</description>
+		</method>
+		<method name="dot" qualifiers="const">
+			<return type="int" />
+			<argument index="0" name="with" type="Vector3i" />
+			<description>
+				Returns the dot product of this vector and [code]b[/code]. This can be used to compare the angle between two vectors. For example, this can be used to determine whether an enemy is facing the player.
```

Comment:
```suggestion
				Returns the dot product of this vector and [code]with[/code]. This can be used to compare the angle between two vectors. For example, this can be used to determine whether an enemy is facing the player.
```